usaspending_api/references/reference_helpers.py

```diff
@@ -5,6 +5,10 @@
 from usaspending_api.accounts.models import FederalAccount, TreasuryAppropriationAccount
 
 
+def remove_federal_accounts(tas_tuple=None):
```

Comment:
Maybe renaming it to `remove_empty_federal_accounts` would clarify things, as well :)